### PR TITLE
Removing heartbeat effect from homepage

### PR DESCRIPTION
**Pull request by mattgperry**
## This PR contains:
Removed heartbeat effect (animations and sound) from homepage

## Describe the problem you have without this PR
The heartbeat effect is quite stressful. It distracts from the messaging of the library, which otherwise looks great!

### CODE REVIEW

**Comment by robertpenner** [comment]
Thank you; I was going to submit the same PR.

File: docs-src/index.html [hunk 1]
```diff
@@ -327,7 +326,7 @@ <h2>Online <b class="underline">is optional</b></h2>
                 Next.js
             </div>
             <div
-                class="neumorphism-circle-m circle centered beating-second"
+                class="neumorphism-circle-m circle centeredd"
```
Comment (keichinger) [inline docs-src/index.html:329]: There's a small typo here.

```suggestion
                class="neumorphism-circle-m circle centered"
```

**Review by keichinger — APPROVED** [review]

**Comment by pubkey** [comment]
Hi.
Sorry for the late answer.
The beating and sound is now disabled by default ac62012691345cb75fe2fbcf358cfe761541de5f

**Comment by pubkey** [comment]
Ok I reverted this now.
I know that playing sound is annoying and people might not like the blinking effect etc.
The problem is that just removing that leave the current landingpage in a state that just looks broken and makes no sense at all.
In the future there might be a full redesign with a page that works without that, but for now I rather have an annoying page then a broken one.

**Comment by pubkey** [comment]
I now added a check to not show the animations to users with `prefers-reduced-motion` set.
This is a temporary fix until we have a new reworked landingpage.
https://github.com/pubkey/rxdb/commit/616928828d829b1925ed29f586e5c2307e716408

**Pull request closed without merging**